workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectWorldCommandSystem.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Collections;
+using Unity.Entities;
+
+#pragma warning disable 649
```

Comment:
What does this do?

Reply:
that's the thing that Jared added, isn't it?
though it should probably be:
```#region Diagnostic control

#pragma warning disable 649
// ReSharper disable UnassignedReadonlyField
// ReSharper disable UnusedMember.Global

#endregion
```

Reply:
`*System.cs` will spawn a load of "variable never assigned/used" warnings because they have injected properties.
This block also disables ReSharper's similar class of warnings for those injected fields. Rather than manually surrounding each field with these, I thought the pragmatic way to go was to turn it off for the whole file, with the knowledge that it's a relatively minor warning in the grand scheme of things.

Reply:
Cool. In which case can we have a comment here saying what it's for?

Reply:
Will do